docs/markdown/Style-guide.md

```diff
@@ -9,7 +9,11 @@ Meson build files.
 
 ## Tabs or spaces?
 
-Always spaces.
+Always spaces. Two spaces is most common.
```

Comment:
Is it really?

I religiously avoid them, personally.

Reply:
Anecdotally, `muon fmt` which is supported by the vscode extension, defaults to 4 spaces but allows you to configure an arbitrary "indentation string".

Reply:
Most of our documentation uses two space indent, and that's what I use, so anything I've worked on has two space indent.